dictionaries/perun.definition.json

```diff
@@ -1,15 +1,15 @@
 {
-  "choose-vo-and-group-tpl_header": {
-    "en": "Access forbidden",
-    "cs": "Přístup odepřen"
+  "choose-vo-and-group-tpl_header1": {
+    "en": "Access forbidden to the service ",
```

Comment:
replace with text: "Access to the service "

Reply:
don't use variable names like header1 header2, instead use header, subheader, etc.

Reply:
Solved. I used choose-vo-and-group-tpl_header-part1 and choose-vo-and-group-tpl_header-part2